suzieq/engines/pandas/arpnd.py

```diff
@@ -8,6 +8,12 @@ class ArpndObj(SqPandasEngine):
     def table_name():
         return 'arpnd'
 
+    def _check_ipvers(self,  addr: pd.Series, version: int) -> list:
+
+        return addr.apply(lambda a: (
+            True if self._get_ipvers(a) == version else False)
+            )
```

Comment:
It might be worth to move this utility function to `engineobj.py`. I-m not sure if it will be useful in other classes.

Reply:
OK